.gitignore

```diff
@@ -7,3 +7,5 @@ environment*
 data/geojsons/*
 
 *.DS_Store
+
+/__pycache__
```

Comment:
This change should really be in a separate commit from the README updates. The leading slash should also be removed so that `__pycache__` directories anywhere are ignored, not just the top-level one. Cached compiled Python modules should never be committed to version control.